bigbluebutton-html5/imports/api/users/server/handlers/validateAuthToken.js

```diff
@@ -45,7 +45,8 @@ export default function handleValidateAuthToken({ payload }) {
         addWelcomeChatMessage(meetingId, userId);
       }
 
-      return Logger.info(`Validated auth token as '${validStatus}' user=${userId} meeting=${meetingId}`);
+      return Logger.info(`Validated auth token as '${validStatus}'
+        user=${userId} meeting=${meetingId}`);
```

Comment:
we use template literals here, not string concatenation..
doesn't it treat a new line and additional spaces as a part of the new string?
If it does, then a one line log message will become a 2 line log message with additional spaces.

You do a similar change in multiple places below, we should really double check that.